package.json

```diff
@@ -18,7 +18,8 @@
     "jest": "^27.5.1"
   },
   "dependencies": {
-    "object-hash": "^1.1.7"
+    "object-hash": "^1.1.7",
+    "yaml": "^2.3.4"
```

Comment:
**NOTE:** This new dependency is needed to be able to export the documentation preview in YAML format.